samples/rag/csharp-inproc/demo.http

```diff
@@ -0,0 +1,27 @@
+### Ingest first email into database
+
+POST http://localhost:7071/api/IngestEmail
+Content-Type: application/json
+
+{"FilePath":""}
```

Comment:
Should we put placeholder values here? That might make it more obvious to users that they need to put their own values for `"FilePath"`.